File: ios/KotlinMultiplatformIos/KotlinMultiplatformIos/PlatformLogger.swift
//
//  PlatformLogger.swift
//  KotlinMultiplatformIos
//
//  Created by V-Mobile on 10/16/18.
//  Copyright © 2018 V-Mobile. All rights reserved.
//

import Foundation
import SharedCode

class PlatformLogger:Logger {
    func log(message: String?) {
        print(message)
    }
}

File: android/app/src/main/java/io/github/armcha/kotlinmultiplatformexample/util/PlatformLogger.kt
package io.github.armcha.kotlinmultiplatformexample.util

import android.util.Log
import org.kotlin.mpp.mobile.Logger


/**
 *
 * Created by Arman Chatikyan on 17 Oct 2018
 *
 */

class PlatformLogger : Logger {

    override fun log(message: String?) {
        Log.e("Tag", message)
    }
}